src/show/Application.js

```diff
@@ -6,8 +6,8 @@ import { Navbar } from '../show/components';
 const Application = () => {
   return (
     <div className='flex flex-col justify-between min-h-screen h-screen w-screen'>
-      <Navbar />
       <BrowserRouter>
```

Comment:
Move `BrowserRouter` to be the first child in this return.

Reply:
ok